src/popup-prompt/main.scss

```diff
@@ -6,43 +6,36 @@
 }
 
 .subscription-prompt--flag {
-	background-color: oColorsGetPaletteColor('black');
+	background-color: oColorsGetPaletteColor('mandarin');
 	border-radius: 3px;
-	padding: 4px 9px 5px;
+	padding: 4px 12px 5px;
 	text-align: center;
-	color: white;
+	color: oColorsGetPaletteColor('black-80');
 	font-size: 15px;
```

Comment:
Could we summarise these typography properties (size, weight, line-height) into an oTypographySansBold($scale:0, $line-height: 17px, $progressive:true) mixin? http://codedocs.webservices.ft.com/v1/sassdoc/o-typography@v5.0.3/#mixin-oTypographySansBold